src/Nox.ClientApi.Tests/Tests/Controllers/CountriesControllerTests.cs

```diff
@@ -1896,6 +1897,333 @@ public async Task WhenGetEnumerationValuesNotExist_ShouldGetTranslatedNameInBrac
+            continentTranslations.Should().Contain(x => x.Name == "Asia" && x.CultureCode == "en-US");
+            
+            var continents = (await GetODataCollectionResponseAsync<IEnumerable<CountryContinentDto>>($"{Endpoints.CountriesUrl}/CountryContinents"))?.ToList();
+            
+            continents.Should().NotBeNull();
+            continents!.Count.Should().Be(5);
+            continents.Should().Contain(x => x.Name == "Asia");
+            
+        }
+
+        [Fact]
+        public async Task WhenUpdateDefaultEnumerationTranslations_ShouldUpdateTranslationsAndDefaults()
+        {
+            // initial
+            await PutNewCountryContinents();
+            
+            
+            var initialSet = (await GetODataCollectionResponseAsync<IEnumerable<CountryContinentLocalizedDto>>(Endpoints.CountriesUrl + "/CountryContinentsLocalized"))?.ToList();
+            
+            
+            initialSet.Should().NotBeNull();
+            initialSet!.Count.Should().Be(10);
+            initialSet.Should().Contain(x => x.Name == "Afrique" && x.CultureCode == "fr-FR");
+            initialSet.Should().Contain(x => x.Name == "Africa" && x.CultureCode == "en-US");
+            
+            // Arrange 
+            var continentLocalizedDtos =  new[] //en-US
+            {
+                new CountryContinentLocalizedDto() { Id = 1, Name = "Europe", CultureCode = "en-US" },
+                new CountryContinentLocalizedDto() { Id = 2, Name = "Asiaa", CultureCode = "en-US" },
+                new CountryContinentLocalizedDto() { Id = 3, Name = "Africa", CultureCode = "en-US" },
+                new CountryContinentLocalizedDto() { Id = 4, Name = "America", CultureCode = "en-US" },
+                new CountryContinentLocalizedDto() { Id = 5, Name = "Oceania", CultureCode = "en-US" }
+            };
+            
+            // Act
+            await PutNewCountryContinents(continentLocalizedDtos);
+            
+            // Assert
+            
+            var continentTranslations = (await GetODataCollectionResponseAsync<IEnumerable<CountryContinentLocalizedDto>>(Endpoints.CountriesUrl + "/CountryContinentsLocalized"))?.ToList();
+
+            continentTranslations.Should().NotBeNull();
+            continentTranslations!.Count.Should().Be(10);
+            continentTranslations.Should().Contain(x => x.Name == "Asie" && x.CultureCode == "fr-FR");
+            continentTranslations.Should().Contain(x => x.Name == "Asiaa" && x.CultureCode == "en-US");
+            
+            var continents = (await GetODataCollectionResponseAsync<IEnumerable<CountryContinentDto>>($"{Endpoints.CountriesUrl}/CountryContinents"))?.ToList();
+            
+            continents.Should().NotBeNull();
+            continents!.Count.Should().Be(5);
+            continents.Should().Contain(x => x.Name == "Asiaa");
+
+        }
+
+        [Fact]
+        public async Task WhenDeleteEnumerations_ShouldRemoveTranslations()
```

Comment:
You are not deleting Enumerations here, but you are deleting translations for certain Enumeration attributes.

Reply:
Desired changes have been applied